### PR TITLE
Add item highlighter and baritone pause module

### PR DESCRIPTION
**Pull request by LordMZTE**
Let the change requests rain!

### CODE REVIEW

**Comment by zeroeightysix** [comment]
Please separate baritone pause module and item highlighter into two PRs :)

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    @EventHandler
+    val onTick = Listener<TickEvent.InGame>({
+        tryInitProcess()
+    })
+
+    private fun tryInitProcess() {
+        if (registeredProcess) return
+        BaritoneIntegration {
+            val mngr = BaritoneAPI.getProvider()?.primaryBaritone?.pathingControlManager
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt:38]: use the baritone utils variable for this?

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        process?.process?.isPaused = false
+    }
+
+    override fun onEnable() {
+        process?.process?.isPaused = true
+    }
+}
+
+// this wrapper is required in order to keep kami from crashing without baritone
+class BaritoneData(val process: PauseBaritoneProcess) {
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt:56]: better name for this, such as BaritoneWrapper

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+    class PauseBaritoneProcess(
+        val displayName: String,
+        var isPaused: Boolean = false
+    ) : IBaritoneProcess {
+        override fun isActive(): Boolean = isPaused
+
+        override fun onTick(p0: Boolean, p1: Boolean) =
+            PathingCommand(null, PathingCommandType.REQUEST_PAUSE)
+
+        override fun isTemporary() = true
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt:66]: ... no?

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+
+// this wrapper is required in order to keep kami from crashing without baritone
+class BaritoneData(val process: PauseBaritoneProcess) {
+    class PauseBaritoneProcess(
+        val displayName: String,
+        var isPaused: Boolean = false
+    ) : IBaritoneProcess {
+        override fun isActive(): Boolean = isPaused
+
+        override fun onTick(p0: Boolean, p1: Boolean) =
```
Comment (5HT2) [inline src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt:63]: oh god please use a proper pause process like kami blue does

File: src/main/resources/kami.mixins.json [hunk 5]
```diff
@@ -65,6 +65,7 @@
     "MixinRenderPlayer",
     "MixinRenderTickCounter",
     "MixinStriderEntity",
-    "MixinWorldRenderer"
+    "MixinWorldRenderer",
+    "MixinHandledScreen"
```
Comment (5HT2) [inline src/main/resources/kami.mixins.json:69]: alphabetical order please

**Review by 5HT2 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt (hunk 3 above)
Comment (LordMZTE) [reply src/main/kotlin/me/zeroeightsix/kami/feature/module/BaritonePause.kt:66]: thats exactly what the pause command does

**Comment by LordMZTE** [comment]
please submit the reviews to the new PR that ill submit soon

**Comment by 5HT2** [comment]
Yea, I wrote the review before it was closed lol

**Comment by LordMZTE** [comment]
can you please submit this again on the new pr? that would make it less confusing

**Pull request closed without merging**